broker/pom.xml

```
@@ -80,13 +80,66 @@
             <version>${netty.version}</version>
         </dependency>
 
+
         <dependency>
-            <groupId>io.netty</groupId>
-            <artifactId>netty-transport-native-epoll</artifactId>
+          <groupId>io.netty</groupId>
+          <artifactId>netty-transport-native-epoll</artifactId>
+            <version>${netty.version}</version>
+          <classifier>linux-x86_64</classifier>
```

```suggestion
          <groupId>io.netty</groupId>
          <artifactId>netty-transport-native-epoll</artifactId>
          <version>${netty.version}</version>
          <classifier>linux-x86_64</classifier>
```

Align all `version` xml tags, in this use and all the other newly introuced

---

Fixed the indent

---

also updated the change log